package.json

```
@@ -66,7 +75,10 @@
         "tabWidth": 4
     },
     "lint-staged": {
-        "*.{html,md,less,json,yml,js,ts,tsx}": "prettier --write"
+        "*.{html,md,less,json,yml,js,ts,tsx}": [
```

```suggestion
        "*.{html,md,less,json,yml,js,mjs,ts,tsx}": [
```
`tsconfig.json` 里也开一下 `checkJs`，并 
`include` `*.mjs`。